docker-compose.local.yml

```diff
@@ -8,9 +8,9 @@ services:
     ports:
       - 5432:5432
     environment:
-      - POSTGRES_USER=hello_django
```

Comment:
Let's put it to the .env file, we have some of the environment variables here some there.

Reply:
ok